bgpd/bgp_bmp.c

```diff
@@ -1314,19 +1340,19 @@ static bool bmp_wrsync(struct bmp *bmp, struct pullwr *pullwr)
 			    bpi_num_labels);
 	}
 
+	peer_type_flag = bmp_get_peer_type(bpi->peer);
```

Comment:
`bpi` can be NULL here, we should check it before dereferencing it.

Reply:
I protected it.